### ISSUE TITLE
dotnet build -v   without specifing verbosity level throws exception

### ISSUE THREAD

**Issue by krwq** [dotnet/cli#5295]
`-v` switch alone is pretty popular, since we require passing verbosity level here (i.e. `-v d`) we should at least produce some good error message

## Steps to reproduce
```
dotnet build -v

Unhandled Exception: System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at Microsoft.DotNet.Cli.CommandLine.CommandLineApplication.ParseOption(Boolean isLongOption, CommandLineApplication command, String[] args, Int32& index, CommandOption& option)
   at Microsoft.DotNet.Cli.CommandLine.CommandLineApplication.Execute(String[] args)
   at Microsoft.DotNet.Tools.Build.BuildCommand.Run(String[] args)
   at Microsoft.DotNet.Cli.Program.ProcessArgs(String[] args, ITelemetry telemetryClient)
   at Microsoft.DotNet.Cli.Program.Main(String[] args)
```

## Expected  behavior
reasonable error

## Actual behavior
exception

## Environment data
`dotnet --info` output:

```
dotnet --info
.NET Command Line Tools (1.0.0-preview5-004506)

Product Information:
 Version:            1.0.0-preview5-004506
 Commit SHA-1 hash:  cf8fd7dd34

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.14393
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   D:\cli\artifacts\win10-x64\stage2\sdk\1.0.0-preview5-004506
```

**Comment by blackdwarf** [comment]
Agreed. We should have at least an error that states that the parameter value was not specified. I would think `CommandLineApplication` has this, right?

**Comment by blackdwarf** [comment]
I believe this has been merged and is now fixed, no?

**Comment by jonsequitur** [comment]
Yes, it has.

**Comment by blackdwarf** [comment]
Closing (fix in #5577).
